diff --git a/configure b/configure
index 2476dbf..cd135da 100755
--- a/configure
+++ b/configure
@@ -4861,7 +4861,8 @@ program_transform_name=`echo $program_transform_name | sed -f conftest.sed`
 rm conftest.sed
 
 # expand $ac_aux_dir to an absolute path
-am_aux_dir=`cd $ac_aux_dir && pwd`
+#am_aux_dir=`cd $ac_aux_dir && pwd`
+am_aux_dir=`cd $ac_aux_dir && pwd | sed "s: :\\\\\\ :g"`
 
 test x"${MISSING+set}" = xset || MISSING="\${SHELL} $am_aux_dir/missing"
 # Use eval to expand $SHELL
@@ -5786,16 +5787,18 @@ echo "$as_me: error: Cannot find the BuildTools directory" >&2;}
 fi
 
 # helpful variable for the base directory of this package
-abs_source_dir=`cd $srcdir; pwd`
+#abs_source_dir=`cd $srcdir; pwd`
+abs_source_dir=`cd $srcdir; pwd | sed "s: :\\\\\\ :g"`
 
 
 # Stuff for example Makefiles
-if test x$prefix = xNONE; then
+if test "x$prefix" = "xNONE"; then
   full_prefix=$ac_default_prefix
 else
   full_prefix=$prefix
 fi
-full_prefix=`cd $full_prefix ; pwd`
+#full_prefix=`cd $full_prefix ; pwd`
+full_prefix=`cd $full_prefix ; pwd | sed "s: :\\\\\\ :g"`
 
 abs_lib_dir=$full_prefix/lib
 
@@ -20656,7 +20659,7 @@ if test $enable_shared = yes; then
     *-linux-*)
       if test "$GXX" = "yes"; then
         RPATH_FLAGS=
-        for dir in $abs_lib_dir; do
+        for dir in "$abs_lib_dir"; do
           RPATH_FLAGS="$RPATH_FLAGS -Wl,--rpath -Wl,$dir"
         done
       fi ;;
@@ -20673,7 +20676,7 @@ if test $enable_shared = yes; then
         RPATH_FLAGS=nothing ;;
     *-*-solaris*)
         RPATH_FLAGS=
-        for dir in $abs_lib_dir; do
+        for dir in "$abs_lib_dir"; do
           RPATH_FLAGS="$RPATH_FLAGS -R$dir"
         done
   esac
@@ -22091,7 +22094,8 @@ if test x$coin_skip_ac_output != xyes; then
     echo $coin_subdirs > coin_subdirs.txt
   else
     # substitute for OBJDIR, needed to setup .pc file for uninstalled project
-    ABSBUILDDIR="`pwd`"
+    #ABSBUILDDIR="`pwd`"
+    ABSBUILDDIR="`pwd | sed "s: :\\\\\\ :g"`"
 
   fi
 
@@ -23443,7 +23447,7 @@ if test "$no_recursion" != yes; then
 
   # Always prepend --prefix to ensure using the same prefix
   # in subdir configurations.
-  ac_sub_configure_args="--prefix=$prefix $ac_sub_configure_args"
+  ac_sub_configure_args="--prefix=\"$prefix\" $ac_sub_configure_args"
 
   ac_popdir=`pwd`
   for ac_dir in : $subdirs; do test "x$ac_dir" = x: && continue
